locales/bn/bn.json

```diff
@@ -628,59 +628,59 @@
+    "United States Outlying Islands": "মার্কিন যুক্তরাষ্ট্র আউটলাইং দ্বীপপুঞ্জ",
     "Update": "আপডেট",
-    "Update & Continue Editing": "আপডেট & সম্পাদনা চালিয়ে যাও",
+    "Update & Continue Editing": "আপডেট & সম্পাদনা চালিয়ে যান",
     "Update :resource": "আপডেট :resource",
     "Update :resource: :title": "আপডেট :resource: :title",
     "Update attached :resource: :title": "সংযুক্ত আপডেট :resource: :title",
     "Update Password": "পাসওয়ার্ড আপডেট করুন",
-    "Update Payment Information": "Update Payment Information",
-    "Update Payment Method": "Update Payment Method",
-    "Update your account's profile information and email address.": "আপনার অ্যাকাউন্ট এর প্রোফাইল তথ্য এবং ইমেল ঠিকানা আপডেট করুন.",
-    "Uruguay": "সোয়াজিল্যাণ্ড",
+    "Update Payment Information": "পেমেন্ট তথ্য আপডেট করুন",
+    "Update Payment Method": "পেমেন্ট মেথড আপডেট করুন",
+    "Update your account's profile information and email address.": "আপনার অ্যাকাউন্ট এর প্রোফাইল তথ্য এবং ইমেল ঠিকানা আপডেট করুন।",
+    "Uruguay": "উরুগুয়ে",
     "Use a recovery code": "একটি পুনরুদ্ধারের কোড ব্যবহার করুন",
     "Use an authentication code": "প্রমাণীকরণ কোড ব্যবহার করো",
-    "Uzbekistan": "সুইডেন",
+    "Uzbekistan": "উজবেকিস্তান",
     "Value": "মান",
-    "Vanuatu": "সুইজারল্যাণ্ড",
-    "VAT Number": "VAT Number",
-    "Venezuela": "তাইওয়ান",
-    "Venezuela, Bolivarian Republic of": "Venezuela, Bolivarian Republic of",
+    "Vanuatu": "ভানুয়াতু",
+    "VAT Number": "ভ্যাট নাম্বার",
+    "Venezuela": "ভেনেজুয়েলা",
+    "Venezuela, Bolivarian Republic of": "ভেনেজুয়েলা, বলিভারিয়ান প্রজাতন্ত্র",
     "Verify Email Address": "ই-মেইল ঠিকানা যাচাই করুন",
-    "Verify Your Email Address": "Verify Your Email Address",
-    "Viet Nam": "Vietnam",
+    "Verify Your Email Address": "আপনার ই-মেইল ঠিকানা যাচাই করুন",
+    "Viet Nam": "ভিয়েতনাম",
     "View": "প্রদর্শন",
-    "View Receipt": "View Receipt",
+    "View Receipt": "রিসিপ্ট দেখুন",
     "Virgin Islands, British": "ব্রিটিশ ভার্জিন দ্বীপপুঞ্জ",
     "Virgin Islands, U.S.": "মার্কিন ভার্জিন দ্বীপপুঞ্জ",
-    "Wallis And Futuna": "আল্লাহর নাম ও সিফাতের তাওহীদ",
-    "Wallis and Futuna": "Wallis and Futuna",
+    "Wallis And Futuna": "ওয়ালিস এবং ফুটুনা",
+    "Wallis and Futuna": "ওয়ালিস এবং ফুটুনা",
     "We are processing your subscription. Once the subscription has successfully processed, this page will update automatically. Typically, this process should only take a few seconds.": "We are processing your subscription. Once the subscription has successfully processed, this page will update automatically. Typically, this process should only take a few seconds.",
     "We are unable to process your payment. Please contact customer support.": "We are unable to process your payment. Please contact customer support.",
     "We were unable to find a registered user with this email address.": "আমরা এই ইমেল ঠিকানা সহ রেজিস্টার্ড ব্যবহারকারী খুঁজে পেতে অসমর্থ ছিল.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.